tickets/src/actions/ticket.js

```diff
@@ -20,7 +20,8 @@ export const signAddress = address => ({
   address,
 })
 
-export const gotSignedAddress = signedAddress => ({
+export const gotSignedAddress = (address, signedAddress) => ({
```

Comment:
API change here, in order to add to the state we need a key. Makes sense to associate the signed address with the original address, so that's what I've done.